examples/gcp_cli.py

```diff
@@ -96,7 +96,18 @@ def QueryLogs(args: 'argparse.Namespace') -> None:
     args (argparse.Namespace): Arguments from ArgumentParser.
   """
   logs = gcp_log.GoogleCloudLog(args.project)
-  results = logs.ExecuteQuery(args.filter)
+
+  qfilter = ''
+  if args.start:
+    qfilter += 'timestamp>=\"{0:s}\" '.format(args.start)
+  if args.start and args.end:
+    qfilter += 'AND '
+  if args.end:
+    qfilter += 'timestamp<=\"{0:s}\" '.format(args.end)
+  if args.filter:
+    qfilter += args.filter
```

Comment:
Don't you need a check to make sure that the two filters play well together? Like an `AND` inserted there somewhere?

Reply:
Line 103/104 contain the 'AND' logic.

Reply:
From what I see `AND` is only inserted in between the two timestamps query, but not in between the timestamp query and the `filter` part. So if you specify `start` and `end` and `filter`, you'll end up with something like `start AND end filter` I would expect `start AND end AND filter`.